internal/common/utilities.go

```diff
@@ -83,3 +83,24 @@ func YmlMarshallTestHelper(in interface{}, t *testing.T) string {
 	assert.NoError(t, err)
 	return string(out)
 }
+
+func SplitBefore(s, sep string) []string {
```

Comment:
Analogous to `strings.SplitAfter` but preserves the splitter at the beginning of the segment.  Useful since we inject a header comment at the beginning of each file when we write them to the post-renderer.  Tests included.